src/views/OverKeyboardView/index.tsx

```diff
@@ -20,10 +20,20 @@ import type { PropsWithChildren } from "react";
  * </OverKeyboardView>
  * ```
  */
-const OverKeyboardView = (props: PropsWithChildren<OverKeyboardViewProps>) => {
-  const { children, visible } = props;
-  const { height, width } = useWindowDimensions();
-  const inner = useMemo(() => ({ height, width }), [height, width]);
+const OverKeyboardView = ({
+  children,
+  visible,
+  height: propHeight,
+  width: propWidth,
+}: PropsWithChildren<OverKeyboardViewProps>) => {
```

Comment:
Let's not use destruction here, because jsdoc plugin will require to write a lot of jsdoc 😅